kotlin-analysis-gradle/src/main/kotlin/org/jetbrains/research/ml/kotlinAnalysis/gradle/GradleFileUtil.kt

```diff
@@ -32,17 +32,19 @@ class GradleFileUtil {
                         ?: GRADLE_DEPENDENCIES_FULL_REGEX.matchEntire(dependencyLine))
                         ?.groups
                         ?.let {
-                            val key = it[1]?.value ?: return null
-                            val group = it[2]?.value ?: return null
-                            val name = it[3]?.value ?: return null
-                            Triple(key, group, name)
+                            val configKey = it[1]?.value ?: return null
+                            val groupId = it[2]?.value ?: return null
+                            val artifactId = it[3]?.value ?: return null
```

Comment:
Are we sure that we have 3 groups? The same comment for kotlin regex. Maybe should we check the size before this?